tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 3.8.0
-envlist = py{38,39,310},bandit,black,flake8,isort
+envlist = py{38,39,310,311,312},bandit,cleaning
```

Comment:
Doesn’t ruff incorporate bandit tests, too?

Reply:
Does it? I am not too sure.

Reply:
Some of the above warnings are from [ruff’s flake8-bandit emulation](https://docs.astral.sh/ruff/rules/#flake8-bandit-s).